app/main.py

```diff
@@ -2,5 +2,14 @@
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    saved_results = {}
+
+    def wrapper(*args, **kwargs) -> Callable:
+        if args in saved_results:
```

Comment:
* use `functools.wraps`
* what  if the function takes the same `args ` but different `kwargs` 
* you must use `args ` with `kwargs`  as the key of `saved_results`